src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/GitRepositoryLauncher.java

```diff
@@ -196,8 +196,6 @@ public JSAP defineArgs() throws JSAPException {
         opt = new FlaggedOption("gitbranch");
         opt.setLongFlag("gitbranch");
         opt.setStringParser(JSAP.STRING_PARSER);
-        opt.setDefault("master");
```

Comment:
we have both gitrepobranch in /GitRepositoryLauncherUtils.java and gitbranch here. That's confusing. Could we unify?

Reply:
Ah yes, this option is due to the Jenkins_Plugin mode which is now removed.  Perhaps I can remove it in the next PR as to address this issue (#1081) ? :)